processors/config/recoTuple_cfg.py

```diff
@@ -69,7 +69,7 @@
 track.parameters["trkRelCollLcio"] = 'TrackDataRelations'
 track.parameters["trkhitCollRoot"] = 'RotatedHelicalOnTrackHits'
 track.parameters["hitFitsCollLcio"] = 'SVTFittedRawTrackerHits'
-track.parameters["rawhitCollRoot"] = ''#'SVTRawHitsOnTrack'
+track.parameters["rawhitCollRoot"] = 'SVTRawHitsOnTrack'
```

Comment:
I think this is a remnant of how this ended up all getting merged into the current master. I guess it would make sense for the default to not use them. I changed this back.